notification_center.py

```diff
@@ -26,15 +29,16 @@
 weechat.hook_print('', 'irc_privmsg', '', 1, 'notify', '')
 
 def notify(data, buffer, date, tags, displayed, highlight, prefix, message):
-	if weechat.config_get_plugin('show_highlights') == 'on' and highlight == '1':
+        sound = 'Pong' if weechat.config_get_plugin('sound') == 'on' else lambda:_
```

Comment:
This is just a dirty hack because the `Notifier.notify` call fallbacks to the default sound even if you pass `None`or `''` to the sound parameter, passing a lambda makes no sound.